grass7/imagery/i.sentinel/i.sentinel.mask/i.sentinel.mask.py

```diff
@@ -222,6 +231,8 @@ def main ():
                     gscript.fatal(error_msg)
                 elif a[0] == 'MTD_TL.xml' and not mtd_file:
                     mtd_file = a[1].strip()
+                elif a[0] == 'metadata' and not metadata_file:
+                    metadata_file = a[1].strip()
```

Comment:
Sorry but it's not so clear to me what this lines do when executed. This part was modified by @ninsbl and I'm not sure to fully understand.

Reply:
Have not checked deeply, but my understanding is that when a file with input information is given, this line extracts the path to the metadata file.

Should probably be added to the documentation...

Reply:
I added one sentence in the html

Reply:
Thanks, now it's clearer to me!